MREUnityRuntime/MREUnityRuntimeLib/Assets/Definitions.cs

```diff
@@ -98,8 +103,31 @@ public enum AlphaMode
     }
 
     /// <summary>
-    /// Contains material asset info
+    /// Type  describing how to interpret the video Source URI
     /// </summary>
+    public enum VideoSourceType
```

Comment:
>VideoSourceType [](start = 16, length = 15)

Noting here that this should be removed, to match the SDK implementation.

Reply:
has been removed